spec/stack_master/parameter_resolvers/one_password_spec.rb

```diff
@@ -158,7 +158,7 @@
       it 'we return an error' do
         allow_any_instance_of(described_class).to receive(:`).with("op --version").and_return(true)
         allow_any_instance_of(described_class).to receive(:`).with("op get item --vault='Shared' 'password title' 2>&1").and_return('{key: value }')
-        expect { resolver.resolve(the_password) }.to raise_error(StackMaster::ParameterResolvers::OnePassword::OnePasswordInvalidResponse, /Failed to parse JSON returned, {key: value }: \d+: unexpected token at '{key: value }'/)
+        expect { resolver.resolve(the_password) }.to raise_error(StackMaster::ParameterResolvers::OnePassword::OnePasswordInvalidResponse, /Failed to parse JSON returned, {key: value }:.* unexpected token at '{key: value }'/)
```

Comment:
Be more lenient with the error message expectation here. Different versions of Ruby provide slightly different error messages.